vars/main.yml

```diff
@@ -10,3 +10,7 @@ keycloak_route_name: keycloak
 keycloak_secret_name: keycloak
 keycloak_pv_claim_name: keycloak-metrics
 postgres_secret_name: keycloak-postgres
+
+dashboards_configmap: grafana-dashboards-configmap
+dashboard_filename: keycloak-dashboard.json # name the file will have inside the config map
+dashboard_file: "{{ lookup('file','/opt/ansible/files/keycloak-dashboard.json') }}" # Contents of the dashboard file
```

Comment:
`dashboard_contents` or `dashboard_file_contents` might work better.
I was unsure from reading the definition in https://github.com/aerogear/oc-patch-file-to-configmap-role what the difference between `dashboard_filename` & `dashboard_file` were until I saw this example here.

Reply:
Thanks @david-martin I agree, this name needs to be more explicit. Will update accordingly